components/support/nimbus-fml/src/workflows.rs

```diff
@@ -158,6 +161,63 @@ pub(crate) fn fetch_file(files: &LoaderConfig, nm: &str) -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn validate(cmd: &ValidateCmd) -> Result<()> {
+    let files: FileLoader = TryFrom::try_from(&cmd.loader)?;
+    let cwd = cmd.loader.cwd.to_str().unwrap();
+
+    let filename = &cmd.manifest;
+    let file_path = files.file_path(filename)?;
+    let parser: Parser = Parser::new(files, file_path.clone())?;
+    let mut loading = HashSet::new();
+    let manifest_front_end = parser.load_manifest(&file_path, &mut loading)?;
+
+    println!(
+        "Loaded modules:\n- {}\n",
+        loading
```

Comment:
`loading` only contains the `includes` of this particular module.

Reply:
If we want to show the imports we've loaded, we can do this with `get_intermediate_representation(channel).all_imports`.

Reply:
`all_imports` only includes imported modules, is there any reason we shouldn't update the `loading` parameter of `load_manifest` to have the includes inserted into it as well?

Alternatively, we could add an `includes` param onto  `load_imports` so the includes don't go into the void (they get put on an empty new hashmap that just kinda disappears right now). Then we can chain `all_imports` moduleIds onto the `loading` and collect them into a hashset to print?

Reply:
Out of github discussion: Use includes non-recursively, and list imports (`loading` set and `all_imports`)